login-workflow/src/screens/LoginScreen/LoginScreen.tsx

```diff
@@ -43,7 +43,7 @@ const EMAIL_REGEX = /^[A-Z0-9._%+'-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
  * @category Component
  */
 
-type LoginScreenPropsPublic = Omit<LoginScreenProps, 'passwordValidator'> & { passwordRequiredValidatorText?: string };
```

Comment:
If we are not using `passwordValidator` prop, it is better to Omit. Please revert this changes